src/adapter/src/coord/ddl.rs

```diff
@@ -931,7 +924,8 @@ impl Coordinator {
                 .clone()
                 .into_inline_connection(self.catalog().state()),
             envelope: sink.envelope,
-            as_of,
+            as_of: self.least_valid_read(&id_bundle),
```

Comment:
nit: the comment above is now somewhat removed from where we calculate the `as_of`, could do an `let as_of = ...` above (but also please ignore if you don't have any other changes that re-trigger ci)

Reply:
will fix